tools/train.py

```diff
@@ -152,7 +153,10 @@ def main():
     # build the runner from config
     if 'runner_type' not in cfg:
         # build the default runner
-        runner = Runner.from_cfg(cfg)
+        if 'strategy' in cfg:
+            runner = FlexibleRunner.from_cfg(cfg)
+        else:
+            runner = Runner.from_cfg(cfg)
```

Comment:
Please use `runner_type` in the config file to specify whether to use `FlexibleRunner`, and don't modify the `train.py` if not necessary.

Reply:
``train.py`` has been restored.